### PR TITLE
Diana

### PR DESCRIPTION
**Pull request by GorDi004**

### CODE REVIEW

File: common/components/ApplyAuctionForm/index.tsx [hunk 1]
```diff
@@ -39,7 +39,9 @@ const ApplyAuctionForm: React.FC<PropsType> = ({ isFormDisabled, form }) => {
       <Form.Item
         name="price"
         label="Ціна"
-        rules={[{ required: true, message: 'Please input amount of money!' }]}
+        rules={[
+          { required: true, message: 'Будь ласка, введіть суму грошей!' },
```
Comment (Iangyl) [inline common/components/ApplyAuctionForm/index.tsx:43]: Введіть ціну послуги

File: common/components/CompetitionWorkerCard/index.tsx [hunk 2]
```diff
@@ -34,10 +34,13 @@ const CompetitionWorkerCard: FC<Props> = ({ _id }) => {
             />
           }
         />
-        <p>Ім&apos;я: {user?.name || user?.email}</p>
-        <p>Електронна пошта: {user?.email}</p>
+        <p>Ім`я: {user?.name || user?.email}</p>
```
Comment (Iangyl) [inline common/components/CompetitionWorkerCard/index.tsx:37]: apostrophe

**Review by Iangyl — CHANGES_REQUESTED** [review]

File: common/components/UI/Steps/index.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { Button, message } from 'antd'
+import { useState } from 'react'
+import s from './style.module.scss'
+
+const Steps: React.FC = (props) => {
+  return <div> </div>
```
Comment (Iangyl) [inline common/components/UI/Steps/index.tsx:6]: What is it?

Comment (GorDi004) [reply common/components/UI/Steps/index.tsx:6]: It is future element for Steps.

File: common/components/ApplyAuctionForm/index.tsx (hunk 1 above)
Comment (GorDi004) [reply common/components/ApplyAuctionForm/index.tsx:43]: Ok

**Pull request merged**